pkg/didcomm/packer/anoncrypt/pack_test.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package anoncrypt
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/google/tink/go/keyset"
+	"github.com/stretchr/testify/require"
+
+	ecdhessubtle "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/transport"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/jose"
+	mockprovider "github.com/hyperledger/aries-framework-go/pkg/internal/mock/provider"
+	"github.com/hyperledger/aries-framework-go/pkg/kms"
+	"github.com/hyperledger/aries-framework-go/pkg/kms/localkms"
+	mockkms "github.com/hyperledger/aries-framework-go/pkg/mock/kms"
+	mockstorage "github.com/hyperledger/aries-framework-go/pkg/mock/storage"
+	"github.com/hyperledger/aries-framework-go/pkg/secretlock/noop"
+)
+
+func TestAnoncryptPackerSuccess(t *testing.T) {
+	k := createKMS(t)
+	recipientsKeys, keyHandles := createRecipients(t, k, 10)
+
+	anonPacker := New(newMockProviderWithCustomKMS(k), jose.A256GCM)
+
+	origMsg := []byte("secret message")
+	ct, err := anonPacker.Pack(origMsg, nil, recipientsKeys)
+	require.NoError(t, err)
+
+	msg, err := anonPacker.Unpack(ct)
+	require.NoError(t, err)
+
+	recKey, err := exportPubKeyBytes(keyHandles[0])
+	require.NoError(t, err)
+
+	require.EqualValues(t, &transport.Envelope{Message: origMsg, ToVerKey: recKey}, msg)
+
+	// try with only 1 recipient
+	// TODO uncomment code when CompactSerlize accepts recipientsKeys headers. It currently rejects recipient headers.
```

Comment:
when is this planned? issue number?

Reply:
From what I've seen in go-jose, compactSerialize() does not accept recipient headers. Not sure if they do key wrapping either. I'll discuss with Derek today and come up with a plan.

Reply:
fixed encrypter/decrypter to accept CompactSerialize, this test has been uncommented and is passing now.